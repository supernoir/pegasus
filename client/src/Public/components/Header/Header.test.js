import React from 'react'
import renderer from 'react-test-renderer'
import {Header} from '.'

test('Component -- Header', () => {
	const component = renderer.create(<Header />)
	let tree = component.toJSON()
	expect(tree).toMatchSnapshot()
})
