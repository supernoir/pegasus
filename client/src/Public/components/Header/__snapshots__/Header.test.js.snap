// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component -- Header 1`] = `
<div
  className="MuiBox-root MuiBox-root-2"
>
  <div
    className="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
  >
    <div
      className="MuiBox-root MuiBox-root-30"
    >
      <h2
        className="MuiTypography-root MuiTypography-h2 MuiTypography-colorTextPrimary MuiTypography-alignCenter"
      >
        Example App
      </h2>
      <p
        className="MuiTypography-root MuiTypography-h3 MuiTypography-colorTextSecondary MuiTypography-paragraph MuiTypography-alignCenter"
      >
        An app example to showcase customizing React SUS usage
      </p>
      <p
        className="MuiTypography-root MuiTypography-h5 MuiTypography-colorTextSecondary MuiTypography-paragraph MuiTypography-alignCenter"
      >
        Please take part in our survey to make sure we can improve your experience!
      </p>
    </div>
  </div>
</div>
`;
